src/tools/analyzer/modules/error.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import logging
+
+from sssd import util
+from sssd.source_files import Files
+from sssd.source_journald import Journald
+from sssd.parser import SubparsersAction
+from sssd.parser import Option
+from sssd import sss_analyze
+
+class ErrorAnalyzer:
+    """
+    An error analyzer module, list if there is any error reported by sssd_be
+    """
+    module_parser = None
+    consumed_logs = []
+    print_opts = [
+        Option('--verbose', 'Verbose output', bool, '-v'),
```

Comment:
I don't believe there is a `--verbose` option with error list, so this should be removed.